ubyssey/static/src/styles/modules/events/_related_events.scss

```diff
@@ -0,0 +1,42 @@
+.c-related-events__row {
+  margin-top: 0.75rem;
+  display: block;
+  width: 100%;
+
+  @media($bp-larger-than-mobile) {
+    display: flex;
+    justify-content: center;
+    flex-direction: row;
+    width: 100%;
+
+    .c-event:not(:last-child) {
+      border-bottom: 0;
```

Comment:
Instead of removing the border, I think it would be better to add some vertical space between the events